source/operations/install-deploy-manage/expand-minio-tenant.rst

```diff
@@ -56,85 +55,44 @@ Persistent Volumes
-     --volumes                 16               \
-     --capacity                16Ti             \
-     --storage-class           local-storage    \
-     --namespace               minio-tenant-1
-
-The following table explains each argument specified to the command:
-
-.. list-table::
-   :header-rows: 1
-   :widths: 30 70
-   :width: 100%
-
-   * - Argument
-     - Description
+1) Expand the MinIO Tenant
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   * - :mc-cmd:`minio-tenant-1 <kubectl minio tenant expand TENANT_NAME>`
-     - The name of the MinIO Tenant which the command expands with the new pool.
+#. From the Operator Console view, select the Tenant to open the summary view, then select :guilabel:`Pools`.
+   Click :guilabel:`Expand Tenant`.
 
-   * - :mc-cmd:`~kubectl minio tenant expand --servers`
-     - The number of ``minio`` servers to deploy in the new Tenant Pool across
-       the Kubernetes cluster.
+#. Specify the following information for the new pool:
 
-   * - :mc-cmd:`~kubectl minio tenant expand --volumes`
-     - The number of volumes in the new Tenant Pool. :mc:`kubectl minio`
-       determines the number of volumes per server by dividing ``volumes`` by
-       ``servers``.
+   .. list-table::
+      :header-rows: 1
+      :widths: 30 70
+      :width: 100%
 
-   * - :mc-cmd:`~kubectl minio tenant expand --capacity`
-     - The total capacity of the Tenant Pool. :mc:`kubectl minio` determines the
-       capacity of each volume by dividing ``capacity`` by ``volumes``.
+      * - Field
+        - Description
 
-   * - :mc-cmd:`~kubectl minio tenant expand --storage-class`
-     - .. cond:: not eks
+      * - Number of Servers
+        - The number of servers to deploy in the new Tenant Pool across the Kubernetes cluster.
      
-          Specify the Kubernetes Storage Class the Operator uses when generating Persistent Volume Claims for the Tenant.
-
-          Ensure the specified storage class has sufficient available Persistent Volume resources to match each generated Persistent Volume Claim.
-
-       .. cond:: eks
-
-          Specify the EBS volume type to use for this tenant.
-          The following list is populated based on the AWS EBS CSI driver list of supported :github:`EBS volume types <kubernetes-sigs/aws-ebs-csi-driver/blob/master/docs/parameters.md>`:
+      * - Volume Size
+        - The capacity of each volume in the new Tenant Pool.
+     
+      * - Volumes per Server
+        - The number of volumes for each server in the new Tenant Pool.
 
-          - ``gp3`` (General Purpose SSD)
-          - ``gp2`` (General Purpose SSD)
-          - ``io2`` (Provisioned IOPS SSD)
-          - ``io1`` (Provisioned IOPS SSD)
-          - ``st1`` (Throughput Optimized HDD)
-          - ``sc1`` (Cold Storage HDD)
+      * - Storage Class
+        - Specify the Kubernetes Storage Class the Operator uses when generating Persistent Volume Claims for the Tenant.
+     
+#. Click :guilabel:`Create`.
```

Comment:
I prefer `Select`, but we just need to be consistent either way.
Mongo style allows for `Click` when specifically talking about pressing and releasing a mouse button. So it's fine. It's just the consistency thing for me.

```suggestion
#. Select :guilabel:`Create`.
```